docs/design/kata-nydus-design.md

```diff
@@ -0,0 +1,89 @@
+# Background
```

Comment:
This file needs to be added to `docs/design/README.md` or the CI will consider it "unreferenced" and fail this PR.

Reply:
has done